README.md

```diff
@@ -45,8 +45,20 @@ returns the length of the shortest tour (not the tour itself).
 
 Test your new function; I've provided some basic testing code in `code.test.js`.
 
+ChatGPT helped with simple randomization algorithm for route, checked other students implementations for help figuring out stop criteria (I was overthinking it)
```

Comment:
I did not copy the method of choosing i and k from another students repo. I only looked through peoples code for their stop criteria for the entire implementation and as soon as I saw someone use the same method I had originally wanted to I moved on with my own program. I did not dissect his code any deeper than that. Having gone back and looked through it I feel that his explanation entirely lacks the reasoning for why his chosen values for i and k ensure they are different and the logic behind it. I fully explained why I chose my criteria and why my criteria works as designated by the assignment. As stated in my sources I consulted with chatGPT for a psuedo code level approach of the problem and perhaps the other student did the same resulting in similar implementations. If that also makes my code unacceptable as my own work I understand but I did not copy the way i and k were chosen from another student.

Reply:
All I'm saying is that your stopping criterion and way of choosing i and k are exactly the same as for the other student.

Reply:
As I previously stated similar uses of language models could account for this but I simply did not take my i and k criteria from another student. I only look at other students code when I am stuck and I was at no point stuck on how to pick i and k. The problem provided clearly stated that they needed to be different and avoid repeated work and so worked out a way to do that accordingly. The stop criteria was a lot less clear to me which is why I sought outside resources.
A simple test I just ran of outlining i and k problem in chat GPT yielded this result: 
    for (let i = 0; i < arrayLength - 1; i++) {
        for (let k = i + 1; k < arrayLength; k++) {
            // code here
        }
     }
So at the very least chatGPT finds this to be the most straightforward approach to this problem and I would agree.

Reply:
Are you saying that ChatGPT generated this code?

Reply:
I have used chatGPT in the ways outlined before, typically for either higher level approaches to problems, individual syntax, or specific lines. I do not use chatGPT or other students for generating blocks of code, only for inspiration. I do not remember exactly what the psuedo-code help given by chatGPT looked like but I am saying it is not out of the question that it could have influenced both of our implementations in similar ways. My code is implemented by me, hand typed by me, and hand annotated to explain why I make the choices I make in my code. I feel that the code submitted for this assignment was not an exception to that.